api/src/test/java/org/openmrs/module/ohricore/OHRICoreServiceTest.java

```diff
@@ -1,3 +1,5 @@
+package org.openmrs.module.ohricore;
+
```

Comment:
@smallgod Delete this test you do not need it

Reply:
@ssmusoke this is done!